docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -291,8 +320,6 @@ You can look again at the cassandrarestore helm-test YAML for the start and endi
 
 `kubectl get cassadrarestore helm-test -o yaml`
 
-![Log output from restore operation](k8ssanda-restore-start-end-timestamps-example.png)
-
 ## Next
 
-Learn how to use the Repair Web Interface (Reaper).
+Learn how to use the [Repair Web Interface]({{< ref "/docs/topics/accessing-services/repair/" >}}).
```

Comment:
I think we should change this from `Repair Web Interface` to `Reaper Web Interface` since it is specific to Reaper.

Reply:
Done - thanks